tests/test_rsa_keys.py

```diff
@@ -161,23 +159,34 @@ def test_verify_rsa_pss_sha256(self):
     rsa_scheme = 'rsassa-pss-sha256'
     data = 'The ancients say the longer the salt, the more provable the security'.encode('utf-8')
 
-    private_key = load_pem_private_key(private_rsa.encode('utf-8'),
-        password=None, backend=default_backend())
-    digest = securesystemslib.hash.digest_from_rsa_scheme(rsa_scheme, 'pyca_crypto')
-
-    # Old-style signature: use the hash length as the salt length.
-    old_signature = private_key.sign(data,
-      padding.PSS(mgf=padding.MGF1(digest.algorithm), salt_length=padding.PSS.DIGEST_LENGTH),
-      digest.algorithm)
-
-    # New-style signature: use the automatic salt length.
-    new_signature, _ = securesystemslib.rsa_keys.create_rsa_signature(private_rsa, data)
-
-    # Verify both old-style and new-style signatures.
-    for signature in (old_signature, new_signature):
-      verified = securesystemslib.rsa_keys.verify_rsa_signature(signature, rsa_scheme,
-                                                                public_rsa, data)
-      self.assertTrue(verified)
+    # Default signature: use the digest length as the salt length.
+    def_sig, _ = securesystemslib.rsa_keys.create_rsa_signature(private_rsa, data)
+
+    # "Max" signature: use the maximum salt length.
+    max_sig, _ = securesystemslib.rsa_keys.create_rsa_signature(private_rsa, data,
+                                                                salt_length=padding.PSS.MAX_LENGTH)
```

Comment:
nit: alignment

Reply:
What about the alignment? I wasn't sure what was the convention. Really we need to fix linting as part of #432.

Reply:
I'm not sure what the problem is. There used to be different expectations on the guidelines but it appears to me that now it just tells "align to open parentheses", @adityasaky am I missing sth?

Reply:
I was looking at https://github.com/secure-systems-lab/code-style-guidelines/blob/master/python.md and I guess we should use `from...import...` to avoid this in the first place?

Reply:
I'm a little nervous about that overhaul considering the whole test above uses it  as such. I'd assume we could go a linter pass sometime later?

Reply:
sgtm.